R/zzz.R

```diff
@@ -279,12 +279,12 @@ CategoryTable <- function(
 #' @seealso \code{\link[shiny]{selectInput}}
 #'
 FilterFeatures <- function(features) {
-  return(sort(x = grep(
+  return(grep(
```

Comment:
Why remove the `sort` from `FilterFeatures`?

Reply:
Because then we'd be sorting the whole list alphabetically before selecting first 1000 rather than after selecting the first 1000 (based on variance)

Reply:
There's a way to include custom sort criteria in selectize.opts so FilterFeatures sorting wouldn't matter, but this was just simpler